lib/components/atoms/Video/Video.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+/**
+ *
+ * Video
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Video.style';
+import type { Props } from './types';
+
+const Video = ({
+  width,
+  height,
+  autoPlay,
+  controls,
+  loop,
+  muted,
+  poster,
+  src,
+  children,
+  className,
+  controlsList,
+}: Props): Node => {
+  return (
+    <video
+      width={width}
+      height={height}
+      controls={controls}
+      autoPlay={autoPlay}
+      loop={loop}
+      muted={muted}
+      poster={poster}
+      className={className}
+      controlsList={controlsList}
+    >
+      <source src={src} type={`video/${src.substring(src.lastIndexOf('.') + 1)}`} />
```

Comment:
Can't we take video type also as a prop instead of calculating this from video URL?

Reply:
Yes, we can do but don't want to do any redundant. So to give the more flexibility we can get TYPE from URL.